server/src/main/kotlin/org/javacs/kt/Main.kt

```diff
@@ -7,21 +7,27 @@ import org.eclipse.lsp4j.launch.LSPLauncher
 import org.javacs.kt.util.ExitingInputStream
 import org.javacs.kt.util.tcpStartServer
 import org.javacs.kt.util.tcpConnectToClient
-
 class Args {
     /*
      * The language server can currently be launched in three modes:
      *  - Stdio, in which case no argument should be specified (used by default)
      *  - TCP Server, in which case the client has to connect to the specified tcpServerPort (used by the Docker image)
      *  - TCP Client, in which case the server will connect to the specified tcpClientPort/tcpClientHost (optionally used by VSCode)
      */
-
     @Parameter(names = ["--tcpServerPort", "-sp"])
     var tcpServerPort: Int? = null
+
     @Parameter(names = ["--tcpClientPort", "-p"])
     var tcpClientPort: Int? = null
+
     @Parameter(names = ["--tcpClientHost", "-h"])
     var tcpClientHost: String = "localhost"
+
+    @Parameter(names = ["--tcpDebug"])
+    var tcpDebug: Boolean = false
+
+    @Parameter(names = ["--tracingLog"])
+    var tracingLog: Boolean = false
```

Comment:
I'd probably prefer making this a more general option for passing the log level if that's an option

Reply:
I see, I will make this part to pass string, and if is tracing, I will tracing the function class name